account_account_tag_code/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2016-2017 ACSONE SA/NV
```

Comment:
Change it to
# Copyright 2023 ForgeFlow S.L.

Reply:
Thanks, I used another module as a template and forgot to change it